docs/source/03_tutorial/07_set_up_experiment_tracking.md

```diff
@@ -0,0 +1,157 @@
+```
+
+Again, we will need to ensure that the dataset is also specified as an output on `pipeline.py` under the `data_processing` pipeline, as follows:
+
+```python
+node(
+    func=preprocess_companies,
+    inputs="companies",
+    outputs=["preprocessed_companies", "companies_columns"],
+    name="preprocess_companies_node",
+)
+```
+
+Having set up both datasets, we are now ready to generate our first set of experiment tracking data!
+
+## Generating Run data
+
+One of the beauty of native experiment tracking in Kedro is that all tracked data are generated and stored each time you do a Kedro run. Hence, to generat the data, simply do:
+
+```bash
+Kedro run
+```
+
+After the run completes, under `data/09_tracking`, you will now see 2 folders, `companies_column.json` and `metrics.json`. On performing a Kedro run after setting up the tracking datasets, Kedro will generate a folder under the dataset name for each tracked dataset, with each folder containing another folder named under the timestamp of the run. Each subsequent Kedro run will generate a new folder named after the timestamp of the run, in the same directory of the tracked dataset.
+
+You will also now see the `session_store.db` generated from your first Kedro run after enabling experiment tracking, which is used to store all the generated run metadata, alongside the tracking dataset, to be exposed via the GraphQL endpoint for Kedro-Viz.
+
+![](../meta/images/experiment-tracking_folder.png)
+
+Try to perform Kedro run a few times to generate a larger set of experiment data. You can also play around with setting up different tracking datasets, and check the logged data via the genereated JSON data files.
+
+## Accessing Run data and comparing runs on Kedro-Viz
+
+Here comes the fun part of accessing your run data on Kedro-Viz. Having ensured that you are on >=Kedro-Viz4.1.1 (you can confirm your Kedro-Viz version by running `Kedro info`), run
+
+```bash
+kedro viz
+```
+
+On spinning up Kedro-Viz, you will now see an `experiment tracking` icon on your left. Clicking the icon will bring you the the experiment tracking page (you can also access the page via `http://127.0.0.1:4141/runsList`), where you will now see the set of experiment data generated from your previous Kedro runs, as shown below:
+
+![](../meta/images/experiment-tracking_runsList.png)
+
+You will now be able to access, compare and pin your runs by toggling the `Compare runs` button, as shown below:
+
+![](../meta/images/experiment-tracking_demo.gif)
+
+You can also access a more detailed demo [here](https://kedro-viz-live-demo.hfa4c8ufrmn4u.eu-west-2.cs.amazonlightsail.com/).
+
+The Kedro-Viz team will be adding new features in the coming weeks, such as allowing the editing your run title, adding notes, bookmarking and searching your runs, etc. Definitely keep an eye out on the [Kedro-Viz release page](https://github.com/quantumblacklabs/kedro-viz/releases) for the upcoming releases. 
```

Comment:
I would move this paragraph to the `Feedback` section below and refrain from mentioning features here, just in case we decide not to add somethings.

Reply:
thanks for suggesting this - not sure to move this to the feedback section as this is mainly more of a 'watch this space' kind of mention instead of asking for feedback.

The features mentioned here are confirmed features that will be shipped in the upcoming releases for Kedro-viz hence I think they could be mentioned - will come back to amend this part once those features are shipped in the next release of Kedro-viz.

Reply:
P.S the feedback section is now removed as per Yetu's suggestion, so this will stay as is in this section 😄